### PR TITLE
Setup lerna and npm

### PR DESCRIPTION
**Pull request by broccolini**

### CODE REVIEW

File: packages/thenewblk-css/README.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# CSS black color palettes
```
Comment (jxnblk) [inline packages/thenewblk-css/README.md:1]: Did you split the packages by format and not by palette?

File: lerna.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+  "lerna": "2.0.0-rc.2",
+  "packages": [
+    "packages/*"
+  ],
+  "version": "independent"
```
Comment (jxnblk) [inline lerna.json:6]: This looks good, though I'm not sure why you'd want this repo to be independently versioned

**Comment by jxnblk** [comment]
💥

File: packages/thenewblk-css/README.md (hunk 1 above)
Comment (broccolini) [reply packages/thenewblk-css/README.md:1]: Yes.

File: lerna.json (hunk 2 above)
Comment (broccolini) [reply lerna.json:6]: Because I want to test how it works.

**Pull request merged**